frontend/public/components/dashboards-page/dashboards.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { RouteComponentProps } from 'react-router-dom';
+
+import { OverviewDashboard } from './overview-dashboard/overview-dashboard';
+import { HorizontalNav, PageHeading } from '../utils';
+
+const tabs = [
+  {
+    href: '',
+    name: 'Overview',
+    component: OverviewDashboard,
+  },
+  // TODO add extension point to add other dashboards
```

Comment:
I'd prefer doing this extensibility on a card level, i.e. have a "new card" extension where you can optionally specify the tab under which it should be rendered. If not specified, the card would be rendered under the (default) "Overview" tab.

This is similar to how we're handling perspective vs. nav item association in #1628.

Reply:
yes, I agree. We should keep it consistent.